plasma_cash/client/client.py

```diff
@@ -9,27 +9,28 @@
 
 class Client(object):
 
-    def __init__(self, root_chain, child_chain):
+    def __init__(self, root_chain, child_chain, key):
         self.root_chain = root_chain
         self.child_chain = child_chain
+        self.key = utils.normalize_key(key)
 
-    def deposit(self, amount, depositor, currency):
+    def deposit(self, amount, currency):
+        depositor = utils.privtoaddr(self.key)
```

Comment:
`utils.privtoaddr` is used everywhere. 
Probably we can add another field or use `@property` inside client for `address` and use `self.address` instead?

Reply:
Sure!